src/quacc/recipes/emt/md.py

```diff
@@ -0,0 +1,123 @@
+"""
+Molecular Dynamics recipes for EMT.
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from ase.calculators.emt import EMT
+from ase.md.velocitydistribution import (
+    MaxwellBoltzmannDistribution,
+    Stationary,
+    ZeroRotation,
+)
+
+from quacc import job
+from quacc.runners.ase import Runner
+from quacc.schemas.ase import summarize_md_run
+from quacc.utils.dicts import recursive_dict_merge
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from ase.atoms import Atoms
+
+    from quacc.schemas._aliases.ase import DynSchema
+
+
+@job
+def md_job(
+    atoms: Atoms,
+    maxwell_boltzmann_params: dict[str, Any] | None = None,
+    md_params: dict[str, Any] | None = None,
+    **calc_kwargs,
+) -> DynSchema:
+    """
+    Carry out a Molecular Dynamics calculation.
+
+    !!! Note "Units"
+
+        Quacc does not follow ASE standards for Molecular Dynamics units.
+
+        Quacc ALWAYS uses the following units:
+
+        - Time: femtoseconds (fs)
+        - Pressure: GPa
+        - Temperature: Kelvin (K)
+        - Compressibility: 1/GPa
+
+    !!! Note "Keywords"
+
+        Additionally, Quacc uses the keywords `fix_com` and `fix_rot` instead of
+        `fixcm` and `fixrot` to fix the center of mass and rotation, respectively.
+
+        As of March 2024, ASE still accept deprecated keywords such as:
+
+        - temperature instead of temperature_K
+        - pressure instead of pressure_au
+        - compressibility instead of compressibility_au
+        - dt instead of timestep
+
+        All of these keywords will be accepted by Quacc, but it is recommended to use the
+        new keywords to avoid confusion. Everything will always be converted to the Quacc
+        units mentioned above.
```

Comment:
We probably don't need to mention this since ASE itself will raise a deprecation warning if they are used. Ultimately, it's on the user to use the current ASE names.

Reply:
This can probably be deleted yes